panther-labs/panther
# fix: create a destination without selecting a severity level

## Background

closes #1443

## Changes

- remove frontend form validation

## Testing

- mage test:ci
- create a destination, check CW logs to see an alert will not be delivered.
- set the destination override in the rule to point to the destination without default severity, see that alerts still come in
